src/data/api.js

```diff
@@ -107,8 +107,8 @@ const api = {
         },
         "ff-template": {
             path: "ff-template",
-            noApi: true
             title: "Template",
+            noApi: true  // TODO find out what this is for and remove if not needed
```

Comment:
as far a i can remember its for pages without an API tab

Reply:
`noApi` never gets referenced anywhere. The reason `Template`'s API tab shows a placeholder message is because `api-view.js` handles the failed markdown load itself in the `connectedCallback`.

I think we can safely remove it. At the moment it doesn't do anything anyway. Even giving other pages the `noApi: true` property has no effect at all.